### ISSUE TITLE
Unit test in test_Procrustes.praat fails randomly

### ISSUE THREAD

**Issue by rlaboiss** [praat/praat#495]
One of the unit tests in file dwtest/test_Polynomial.praat fails randomly:

```
$ while true ; do xvfb-run -a praat --run dwtest/test_Procrustes.praat || break ; done
[…]
Error: Script assertion fails in line 110 (false):
   abs(scale - sp) < .eps; scale
Script “/var/data/rlaboiss/debian/PKGS/praat/praat/dwtest/test_Procrustes.praat” not completed.
Praat: script command <<dwtest/test_Procrustes.praat>> not completed.
```
Increasing the tolerance in the assertion statement, like in the attached patch [fix-unit-test-procrustes.txt](https://github.com/praat/praat/files/1385321/fix-unit-test-procrustes.txt), drastically decreases the error rate (I could not trigger it, at least, after a long run of the command above).

Of course, this is not a principled way of coping with the problem, but it is working now for the building of the Debian package praat.

**Comment by davidweenink** [comment]
I reworked the test procedure. Hopefully the chances of failing are much smaller now.

**Comment by rlaboiss** [comment]
Thanks, David.  Just for the record, this is the commit that fixed the problem: 14349976bc8536100973a20e22f8928ba7b39971
